client/src/livingdocs-component-app/app.js

```diff
@@ -291,7 +289,7 @@ export class App {
 
       let selectedItem = this.selectedItems[this.selectedItemIndex];
 
-      qEnv.QServerBaseUrl.then(QServerBaseUrl => {
+      await qEnv.QServerBaseUrl.then(QServerBaseUrl => {
```

Comment:
If we have to `await` `loadDisplayOptions` we could return the promise instead of awaiting here afaik.
Or maybe rewrite the whole function with async/await instead of mixing await with the `then` chain.
It probably would get more readable.

Reply:
@manuelroth do you think it would make sense to rewrite this to use await instead of the `then` chain to make it more readable?

Reply:
Yes I agree. I would like to do this in a separate PR. I think the entire livingdocs-component could use a refactoring. The code flow is very confusing. I would like to discuss this in the monday meeting, whether we want to invest some time into this.

Reply:
👍